src/Services/LightningService.cs

```diff
@@ -1222,8 +1222,11 @@ public async Task<(long?, long?)> GetChannelBalance(Channel channel)
             var chan = result.Channels.FirstOrDefault(x => x.ChanId == channel.ChanId);
             if(chan == null)
                 return (null, null);
+            
+            var htlcsLocal = chan.PendingHtlcs.Where(x => x.Incoming == true).Sum(x => x.Amount);
+            var htlcsRemote = chan.PendingHtlcs.Where(x => x.Incoming == false).Sum(x => x.Amount);
```

Comment:
@markettes There's a chan.UnsettledBalance field with that info

Reply:
Thats true, but you cant get to know in what direction is that unsettled balance flowing